Classes/common/HTTP/CDTURLSessionTask.m

```diff
@@ -18,6 +18,12 @@
 #import "CDTHTTPInterceptor.h"
 #import "CDTLogging.h"
 
+@interface CDTHTTPInterceptorContext ()
+
+@property (nullable, nonatomic, strong) NSURLResponse *response;
```

Comment:
Given you allow someone to return a whole new context object, I'm not sure that you're correct there.